sdk/entra/functions-authentication-events/samples-dev/getConfigurationSetting.ts

```diff
@@ -0,0 +1,29 @@
+// // Copyright (c) Microsoft Corporation.
```

Comment:
Is there any real samples that can be added here?

Reply:
Sample Added... Just not sure on how to reference "@azure/functions"